test/test_luaunit.lua

```diff
@@ -1166,6 +1166,30 @@ TestLuaUnitAssertions = { __class__ = 'TestLuaUnitAssertions' }
         assertFailure(lu.assertItemsEquals, {one=1,two=2,three=3}, {two=2,one=1})
     end
 
+    function TestLuaUnitAssertions:test_assertContains()
+        local t = {3, 'some value', 1, 2}
+
+        assertFailure(lu.assertContains, t, 0)
+        lu.assertContains(t, 1)
+        lu.assertContains(t, 2)
+        lu.assertContains(t, 3)
+        assertFailure(lu.assertContains, t, 4)
+        lu.assertContains(t, 'some value')
+        assertFailure(lu.assertContains, t, 'other value')
+    end
+
+    function TestLuaUnitAssertions:test_assertNotContains()
+        local t = {3, 'some value', 1, 2}
+
+        lu.assertNotContains(t, 0)
+        assertFailure(lu.assertNotContains, t, 1)
+        assertFailure(lu.assertNotContains, t, 2)
+        assertFailure(lu.assertNotContains, t, 3)
+        lu.assertNotContains(t, 4)
+        assertFailure(lu.assertNotContains, t, 'some value')
+        lu.assertNotContains(t, 'other value')
+    end
```

Comment:
Same here : I would like to see also some tests on a non-array table, just to make sure that we are not using ipairs instead of pairs in some places